src/cljs/main/broadfcui/common/flex_utils.cljs

```diff
@@ -4,12 +4,12 @@
     ))
 
 
-(defn flex-box [attrs & children]
+(defn box [attrs & children]
   [:div (utils/deep-merge {:style {:display "flex"}} attrs)
    children])
 
-(defn flex-strut [size]
+(defn strut [size]
   [:div {:style {:flexGrow 0 :flexShrink 0 :flexBasis size}}])
 
-(def flex-spacer
+(def spring
```

Comment:
With this you can specify exactly where you want to have extra space:  `[spring comp1 (strut 10) comp2 spring]` to center two components with a fixed gap between them.